### ISSUE TITLE
LTE: ss

### ISSUE THREAD

**Issue by Gitschubser** [wish7code/openbmap#85]
The value ss (signal strength in dbm) shows strange values when act=LTE.
Values: 911, 910, 894, ...
The rxlev has  always stativ value 97.

**Comment by wish7code** [comment]
Might be a bug in the motorola XT1039 API implementation: from all the obviously wrong readings I found, all were recorded with a motorola XT1039.

Nevertheless also other devices might be affected: there were also suspious patterns with other device manufactors, especially constant values for ss and rxlev fields on LTE networks.

We'll have to keep an eye on this, althought I don't have any clue yet how to deal with this issue.
Basically the client is just using the Android API functions
((CellInfoLte) cell).getCellSignalStrength().getDbm() and 
((CellInfoLte) cell).getCellSignalStrength().getAsuLevel())
